packages/create-sitecore-jss/src/templates/nextjs/sitecore/config/{{appName}}.config

```diff
@@ -145,19 +145,21 @@
     -->
     <layoutService>
       <configurations>
-        <config name="default">
-          <rendering>
-            <renderingContentsResolver>
+        <config name="next-default">
```

Comment:
If this does work, we would need to also set the layout service to use it: https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/nextjs/src/lib/layout-service-factory.ts#L21